app/models/alchemy/attachment.rb

```diff
@@ -35,7 +35,7 @@ class Attachment < ActiveRecord::Base
     has_many :pages, :through => :elements
 
     validates_presence_of :file
-    validates_format_of :file_name, with: /\A[A-Za-z0-9\.\-_]+\z/, on: :update
+    validates_format_of :file_name, with: /\A[A-Za-z0-9\. \-_äÄöÖüÜß]+\z/, on: :update
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, format: value` over `validates_format_of`.